#exposure time
float32 time
---
#0:OK, others:failed
int32 ret
